data/mods/gen4/moves.ts

```diff
@@ -1433,6 +1433,27 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 			return !pokemon.volatiles['choicelock'] && !pokemon.volatiles['encore'];
 		},
 	},
+	snatch: {
+		inherit: true,
+		condition: {
+			duration: 1,
+			onStart(pokemon) {
+				this.add('-singleturn', pokemon, 'Snatch');
+			},
+			onAnyPrepareHitPriority: -1,
+			onAnyPrepareHit(source, target, move) {
+				const snatchUser = this.effectState.source;
+				if (snatchUser.isSkyDropped()) return;
```

Comment:
This would be unnecessary since it's pre-Sky Drop.

Reply:
I saw other code check future gen stuff (specifically in the gen 4 moves files, uproar checks for throat chop), which is why I left the future gen stuff in.

Reply:
I suppose it's fine if someone uses Sky Drop in Gen 4 custom game or smth. It can be left in.